CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 ## [unreleased]
 
+### Added
+
+- Support for QHD displays
```

Comment:
Added `sandstone/ThemeDecorator/screenTypes` value `qhd` to support QHD displays
or
Added `screenTypes` value `qhd` to support QHD displays